start.R

```diff
@@ -218,6 +220,10 @@ if (any(c("--reprepare", "--restart") %in% flags)) {
     }
   }
 
+  if (exists("titletag")) {
```

Comment:
Good catch, probably not. I'll have to think about the correct solution. Why is everything always so complicated? :wink:

Reply:
well, I think for getting an overview on the latest REMIND developments it is important to identify all AMT scenarios. This could also happen in runstatistics and there be added to all AMT scenario names, including the reference scenarios

Reply:
Adding this before you change the `row.names(scenario)` should do the job:
```
for (c in names(path_gdx_list)) {
  scenario[[c]][scenario[[c]] %in% row.names(scenario)] <- paste0(titletag, "-", scenario[[c]][scenario[[c]] %in% row.names(scenario)], "-", add)
}
```
You add the `titletag` to all the cells in the `path_gdx*` cells that contain runs that are started. For all the rest (.gdx paths, runs that are not started but automatically detected), everything stays the same. I'm pretty sure you can write that more elegantly :)

One additional thought: I would prefer to have `titletag` appended to the scenario name, because that would be an awesome feature if you run debug runs or testOneRegi of specific regions, you can simply add this tag, but I think it is more sensible to have `SSP2-Base-debug` etc. runs then all being called as `debug-*`. That is also what Robert desired a while ago in the RSE channel. For AMT, that would also be fine, I think.

Reply:
for AMT runs I would prefer having AMT at the beginning of the title but for all other use cases I agree that the end is the better place. Both is fine for me

Reply:
Agree. Suggestion: the command line parameters support vectors, so you can specify `titletag=AMT,debug` or something like that. If `length(titletag) = 1`, append it, if `length(titletag) = 2`, prepend `titletag[1]` and append `titletag[2]`. Then, AMT runs could use `titletag=,AMT` and the normal user could use it as desired.

Reply:
I think this is unnecessary complexity. We had the `AMT` in the middle of the AMT runs' titles before and nobody complained, so it should be fine to have the `AMT` at the end. So we moved the titletag to the end of the title generally.